appGeo/src/tests/game_tests.spec.ts

```diff
@@ -0,0 +1,231 @@
+import {Game, CIVILIAN, KILLER} from '../app/game/game.component'
```

Comment:
Here CIVILIAN and KILLER should not be capitalized (that calls the enum instead).
Instead, this line of code should be `import {Game, Civilian, Killer} from '../app/game/game.component'`

Reply:
That's the point. This is importing the enumerated constants so that they can be utilized in the tests

Reply:
agh, the exports seem to have been overwritten by #56 getting merged in. Fixed that